src/Sentry/TransactionTracer.cs

```diff
@@ -177,22 +181,15 @@ public IReadOnlyList<string> Fingerprint
     /// <summary>
     /// Initializes an instance of <see cref="Transaction"/>.
     /// </summary>
-    public TransactionTracer(IHub hub, string name, string operation)
-        : this(hub, name, operation, TransactionNameSource.Custom)
-    {
-    }
-
-    /// <summary>
-    /// Initializes an instance of <see cref="Transaction"/>.
-    /// </summary>
-    public TransactionTracer(IHub hub, string name, string operation, TransactionNameSource nameSource)
+    public TransactionTracer(IHub hub, string name, string operation, TransactionNameSource nameSource = TransactionNameSource.Custom)
```

Comment:
This is a binary breaking change. Code compiled against the overload you deleted (without the `TransactionNameSource`) would crash at runtime with `MissingMethodException`.

Like this: https://github.com/getsentry/sentry-xamarin/issues/140

Better to leave the overload until a major version bump.

Reply:
Hm, subtle. Gotta know a bit about how the byte code gets assembled to spot that.

Easy fix though 😉 so will do.